src/Cohesity.Powershell/Scripts/Vlan/Get-CohesityVlan.ps1

```diff
@@ -0,0 +1,80 @@
+#### USAGE ####
+#	********************** Using Function *********************
+#   Get-CohesityVlan
+#   Get-CohesityVlan -allUnderHierarchy false -skipPrimaryAndBondIface false -tenantIds testOrg/
+#   Get-CohesityVlan -allUnderHierarchy false -skipPrimaryAndBondIface true
+#   Get-CohesityVlan -VlanId 0
+###############
+
+function Get-CohesityVlan {
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false)]
```

Comment:
not specific to the review, explore a set concept. You can group certain parameters and make things mandatory / autocomplete